icons/cloudflare/cloudflare-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#F6821F" d="M86.496 83.018c-1.163-1.708-3.126-2.693-5.486-2.805l-44.74-.608a.816.816 0 0 1-.689-.38 1.081 1.081 0 0 1-.109-.832 1.233 1.233 0 0 1 1.054-.835l45.136-.608c5.343-.265 11.159-4.779 13.192-10.315l2.581-7.014c.114-.29.139-.606.072-.91-2.905-13.728-14.644-23.968-28.671-23.968-12.936 0-23.915 8.722-27.839 20.819a12.95 12.95 0 0 0-9.267-2.693c-6.216.645-11.191 5.842-11.811 12.325-.15 1.612-.04 3.239.327 4.816C10.105 70.312 2 78.959 2 89.617c0 .948.075 1.895.184 2.843.072.455.436.798.873.798h82.567c.471 0 .907-.343 1.054-.835l.617-2.313c.763-2.731.473-5.238-.799-7.092z"/><path fill="#FBAD41" d="M101.54 60.178c-.399 0-.835 0-1.234.037-.29 0-.546.228-.655.53l-1.746 6.334c-.761 2.731-.471 5.234.801 7.091 1.163 1.708 3.126 2.693 5.486 2.805l9.523.608c.29 0 .545.153.692.38.144.252.183.551.109.832-.156.463-.57.791-1.057.835l-9.919.608c-5.38.265-11.159 4.775-13.192 10.315l-.729 1.933c-.143.38.109.757.511.757h34.086a.902.902 0 0 0 .873-.683c.6-2.251.906-4.571.91-6.901.001-14.028-10.974-25.481-24.459-25.481"/></svg>
```

Comment:
Color palette here should also be based on the official one. There's no version in their logo kit that doesn't have a wordmark, but I approve of editing it so it's without the wordmark though.

Reply:
Like it indicated in the PR : 
> from the github repo https://github.com/cloudflare/research.cloudflare.com/blob/main/img/cloud.svg

Reply:
Can we perhaps alter the logo-kit version, and use that instead?
Just resize and center this maybe?
![cf-logo-v-rev](https://user-images.githubusercontent.com/43886029/180876386-f8e91c79-08e5-446b-920c-e5262480bbc5.svg)
```svg
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 874.3 393.7" style="enable-background:new 0 0 874.3 393.7;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#FFFFFF;}
	.st1{fill:#F38020;}
	.st2{fill:#FAAE40;}
</style>
<g>
		<polygon class="st0" points="752.9,169.6 710,145 702.6,141.8 527.1,143 527.1,232.1 752.9,232.1 		"/>
		<path class="st1" d="M674.8,223.9c2.1-7.2,1.3-13.8-2.2-18.7c-3.2-4.5-8.6-7.1-15.1-7.4l-123.1-1.6c-0.8,0-1.5-0.4-1.9-1
			c-0.4-0.6-0.5-1.4-0.3-2.2c0.4-1.2,1.6-2.1,2.9-2.2l124.2-1.6c14.7-0.7,30.7-12.6,36.3-27.2l7.1-18.5c0.3-0.8,0.4-1.6,0.2-2.4
			c-8-36.2-40.3-63.2-78.9-63.2c-35.6,0-65.8,23-76.6,54.9c-7-5.2-15.9-8-25.5-7.1c-17.1,1.7-30.8,15.4-32.5,32.5
			c-0.4,4.4-0.1,8.7,0.9,12.7c-27.9,0.8-50.2,23.6-50.2,51.7c0,2.5,0.2,5,0.5,7.5c0.2,1.2,1.2,2.1,2.4,2.1l227.2,0
			c1.3,0,2.5-0.9,2.9-2.2L674.8,223.9z"/>
		<path class="st2" d="M714,144.8c-1.1,0-2.3,0-3.4,0.1c-0.8,0-1.5,0.6-1.8,1.4l-4.8,16.7c-2.1,7.2-1.3,13.8,2.2,18.7
			c3.2,4.5,8.6,7.1,15.1,7.4l26.2,1.6c0.8,0,1.5,0.4,1.9,1c0.4,0.6,0.5,1.5,0.3,2.2c-0.4,1.2-1.6,2.1-2.9,2.2l-27.3,1.6
			c-14.8,0.7-30.7,12.6-36.3,27.2l-2,5.1c-0.4,1,0.3,2,1.4,2h93.8c1.1,0,2.1-0.7,2.4-1.8c1.6-5.8,2.5-11.9,2.5-18.2
			C781.3,175,751.1,144.8,714,144.8"/>
</g>
</svg>
```
I feel the logo-kit colors are the ones that should be used at least. Feel free to voice your opposition if you disagree though.